src/lib/rmt/ssl/new_ca_password_dialog.rb

```diff
@@ -28,9 +28,9 @@ def initialize
     super
 
     @dialog_heading = 'Setting CA private key password'
-    @dialog_label = 'Please set new CA private key password'
-    @password_field_label = 'New CA private key &Password'
-    @password_confirmation_field_label = 'New Password &Again'
+    @dialog_label = 'Please set a password for the CA private key\n' \
+                    'to protect the private key.'
```

Comment:
Maybe just keep it as `Please set a password for the CA private key`?
Otherwise `private key` repeats twice and it's not adding any useful information (everybody probably knows that the passwords are for protection :slightly_smiling_face:)

Reply:
You're right.